joomla/joomla-cms
# Joomla.JText without inline JavaScript

It same as #11671 but for `Joomla.JText`, with 'lazy' loading for the strings.
### Testing Instructions

place in template.js of Isis template next test JavaScript:

``` javascript
alert(Joomla.JText._('JTOGGLE_HIDE_SIDEBAR', 'string not found!!!'));
```

Open the Articles page and you should get alert with translation for 'JTOGGLE_HIDE_SIDEBAR'.

UPD: the pull request depends from #11671, 
It still can be tested stand alone, but #11671 should be merged first
